packages/chakra-ui-core/src/CReset/tests/CReset.test.js

```diff
@@ -0,0 +1,47 @@
+import CReset from '../'
+import { mount } from '@/tests/test-utils'
```

Comment:
Is there a reason you decided to go with `@vue/test-utils`? For testing we're mostly relying on using `@testing-library/vue` like I had earlier mentioned [in this comment](https://github.com/chakra-ui/chakra-ui-vue/pull/287#issuecomment-686874639).

I initially used VTU but we decided in the middle of the project to use the Testing Library because it has a more concise syntax and provides more utilities out of the box.